Polyfills/AbortController/Source/AbortSignal.cpp

```diff
@@ -26,6 +26,19 @@ namespace Babylon::Polyfills::Internal
     {
     }
 
+    void AbortSignal::Abort()
+    {
+        m_aborted = true;
+
+        auto onabort = m_onabort.Value();
+        if (!onabort.IsNull() && !onabort.IsUndefined())
+        {
+            onabort.Call({});
+        }
```

Comment:
Does this work if the reference is empty? It seems like you have unit test for this, but I'm not sure why you don't need to check if a reference is empty.

Reply:
It looks like when checking .IsUndefined(), it calls Type() == napi_undefined which Type() then calls IsEmpty(). IsEmpty() returns true, and because IsEmpty() returned true Type() returns napi_undefined. IsUndefined() seems to catch the empty reference case.

Reply:
Interesting. I'm pretty sure this is implemented differently in the napi/jsi adaptor. I might need to fix that.